specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/examples/CreateCloudServiceWithMultiRole.json

```diff
@@ -0,0 +1,171 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "ConstosoRG",
+    "cloudServiceName": "{cs-name}",
+    "api-version": "2021-03-01",
```

Comment:
To easily identify the delta changes from previous api version, the recommendation is to copy the existing version into new directory structure for first commit and then push new changes in subsequent commits. Please raise a new PR with the suggested recommendation for faster review.

Reply:
Done: https://github.com/Azure/azure-rest-api-specs/pull/13155